scripts/build.sh

```diff
@@ -21,3 +19,9 @@ export CGO_ENABLED=0
 GOOS=linux  GOARCH=amd64 go build -ldflags "${LDFLAGS[*]}" -o "$BIN_DIR/scw-$VERSION-linux-x86_64"  cmd/scw/main.go
 GOOS=darwin GOARCH=amd64 go build -ldflags "${LDFLAGS[*]}" -o "$BIN_DIR/scw-$VERSION-darwin-x86_64" cmd/scw/main.go
 GOOS=windows GOARCH=amd64 go build -ldflags "${LDFLAGS[*]}" -o "$BIN_DIR/scw-$VERSION-windows-x86_64.exe" cmd/scw/main.go
+
+shasum -a 256 \
+  "$BIN_DIR/scw-$VERSION-linux-x86_64" \
```

Comment:
`$BIN_DIR/scw-$VERSION-linux-x86_64` is duplicated on line 19

Should we introduce variables for the 3 paths ?